includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -0,0 +1,284 @@
+							'url'   => 'http://...',
+						],
+						[
+							'title' => 'WooCommerce',
+							'url'   => 'http://...',
+						],
+						[
+							'title' => 'Content Drip',
+							'url'   => 'http://...',
+						],
+					],
+				],
+				[
+					'title' => 'Advanced Features',
+					'items' => [
+						[
+							'title' => 'Interactive Blocks',
+							'url'   => 'http://...',
+						],
+						[
+							'title' => 'Groups & Cohorts',
+							'url'   => 'http://...',
+						],
+						[
+							'title' => 'Quizzes',
+							'url'   => 'http://...',
+						],
+						[
+							'title' => 'Integrations',
+							'url'   => 'http://...',
+						],
+					],
+				],
+			],
+			'help'                        => [
+				// TODO: Replace with real implementation.
+				[
+					'title' => 'Get the most out of Sensei',
+					'items' => [
+						[
+							'title' => 'Sensei Documentation',
+							'url'   => 'http://...',
+							'icon'  => null,
+						],
+						[
+							'title' => 'Support forums',
+							'url'   => 'http://...',
+							'icon'  => null,
+						],
+						[
+							'title'        => 'Create a support ticket',
+							'url'          => null,
+							'extra_action' => [
+								'title' => 'Upgrade to Sensei Pro',
+								'url'   => 'https://...',
+							],
+							'icon'         => 'lock',
+						],
+					],
+				],
+			],
+			'guides'                      => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'How to Sell Online Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Creating Video Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Choose the Right Hosting Provider',
+						'url'   => 'http://...',
+					],
+				],
+				'more_url' => 'http://senseilms.com/category/guides/',
+			],
+			'news'                        => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'Introducing Interactive Videos For WordPress',
+						'date'  => '2022-08-31', // Localized for user.
+						'url'   => 'http://senseilms.com/inroducing-interactive-videos/',
+					],
+					[
+						'title' => 'New Block Visibility, Scheduled Content, and Group Features',
+						'date'  => '2022-08-09', // Localized for user.
+						'url'   => 'http://senseilms.com/conditional-content/',
+					],
+				],
+				'more_url' => 'https://senseilms.com/blog/',
+			],
+			'extensions'                  => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro}.json.
+				[
+					'title'        => 'Sensei LMS Post to Course Creator',
+					'image'        => 'http://senseilms.com/wp-content/uploads/2022/02/sensei-post-to-course-80x80.png',
+					'description'  => 'Turn your blog posts into online courses.',
+					'price'        => 0,
```

Comment:
I think this can be a string instead to avoid eventual problems with floats, so the prefix is already included. WDYT?

The extensions API already does that, for reference:

![image](https://user-images.githubusercontent.com/529864/191876055-ef9acb87-374e-4c0c-9df0-19aab5e7b7d3.png)

Reply:
Well, I was assuming here cents. And localising it would make it difficult for the frontend to identify the "free" scenario which requires a special install button text.

Reply:
hmmm, good point. And not sure how to handle it, to be honest.